The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... ok
selecting dynamic shared memory implementation ... posix
selecting default "max_connections" ... 100
selecting default "shared_buffers" ... 128MB
selecting default time zone ... Etc/UTC
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok


Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data -l logfile start

initdb: warning: enabling "trust" authentication for local connections
initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2025-09-09 01:25:47.869 UTC [49] LOG:  starting PostgreSQL 17.6 (Debian 17.6-1.pgdg13+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 14.2.0-19) 14.2.0, 64-bit
2025-09-09 01:25:47.871 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-09-09 01:25:47.877 UTC [52] LOG:  database system was shut down at 2025-09-09 01:25:47 UTC
2025-09-09 01:25:47.882 UTC [49] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

2025-09-09 01:25:48.108 UTC [49] LOG:  received fast shutdown request
waiting for server to shut down....2025-09-09 01:25:48.110 UTC [49] LOG:  aborting any active transactions
2025-09-09 01:25:48.115 UTC [49] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
2025-09-09 01:25:48.115 UTC [50] LOG:  shutting down
2025-09-09 01:25:48.116 UTC [50] LOG:  checkpoint starting: shutdown immediate
2025-09-09 01:25:48.156 UTC [50] LOG:  checkpoint complete: wrote 925 buffers (5.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.020 s, sync=0.017 s, total=0.042 s; sync files=301, longest=0.002 s, average=0.001 s; distance=4256 kB, estimate=4256 kB; lsn=0/1915968, redo lsn=0/1915968
2025-09-09 01:25:48.164 UTC [49] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-09-09 01:25:48.237 UTC [7] LOG:  starting PostgreSQL 17.6 (Debian 17.6-1.pgdg13+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 14.2.0-19) 14.2.0, 64-bit
2025-09-09 01:25:48.237 UTC [7] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-09-09 01:25:48.237 UTC [7] LOG:  listening on IPv6 address "::", port 5432
2025-09-09 01:25:48.239 UTC [7] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-09-09 01:25:48.244 UTC [65] LOG:  database system was shut down at 2025-09-09 01:25:48 UTC
2025-09-09 01:25:48.249 UTC [7] LOG:  database system is ready to accept connections
2025-09-09 01:30:48.343 UTC [63] LOG:  checkpoint starting: time
2025-09-09 01:30:52.767 UTC [63] LOG:  checkpoint complete: wrote 47 buffers (0.3%); 0 WAL file(s) added, 0 removed, 0 recycled; write=4.415 s, sync=0.003 s, total=4.424 s; sync files=12, longest=0.002 s, average=0.001 s; distance=269 kB, estimate=269 kB; lsn=0/19591A8, redo lsn=0/1959150
2025-09-09 01:47:06.536 UTC [122] ERROR:  relation "users" does not exist at character 107
2025-09-09 01:47:06.536 UTC [122] STATEMENT:  select "id", "email", "password", "full_name", "role", "is_system_admin", "created_at", "updated_at" from "users" where "users"."role" = $1
2025-09-09 01:47:28.632 UTC [124] ERROR:  relation "users" does not exist at character 107
2025-09-09 01:47:28.632 UTC [124] STATEMENT:  select "id", "email", "password", "full_name", "role", "is_system_admin", "created_at", "updated_at" from "users" where "users"."role" = $1
2025-09-09 01:47:32.100 UTC [125] ERROR:  relation "users" does not exist at character 107
2025-09-09 01:47:32.100 UTC [125] STATEMENT:  select "id", "email", "password", "full_name", "role", "is_system_admin", "created_at", "updated_at" from "users" where "users"."role" = $1